filebeat/tests/system/test_multiline.py

```diff
@@ -48,11 +50,13 @@ def test_c_style_log(self):
             path=os.path.abspath(self.working_dir) + "/log/*",
             multiline=True,
             pattern="\\\\$",
-            match="after"
+            match="before"
```

Comment:
@urso any reason that was changed? As it seems like the test or implementation didn't change and it passed before, it probably should not pass anymore now?